inst/@sym/ifourier.m

```diff
@@ -17,31 +17,63 @@
 %% If not, see <http://www.gnu.org/licenses/>.
 
 %% -*- texinfo -*-
-%% @deftypefn  {Function File} {@var{f} =} ifourier (@var{FF}, @var{k}, @var{x})
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{f} =} ifourier (@var{FF}, @var{w}, @var{x})
+%% @deftypefnx {Function File} {@var{f} =} ifourier (@var{FF})
+%% @deftypefnx {Function File} {@var{f} =} ifourier (@var{FF}, @var{x})
 %% Symbolic inverse Fourier transform.
 %%
-%% FIXME: doc, examples
+%% FIXME: doc
 %%
-%% @seealso{ilaplace}
+%% Examples:
+%% @example
+%% syms x w
+%% F = 1/(1+w^2)
+%% ifourier(F)
+%% ifourier(F, w)
+%% ifourier(F, x, w)
+%% @end example
+%%
+%% @seealso{fourier,laplace}
 %% @end deftypefn
 
 %% Author: Colin B. Macdonald
 %% Keywords: symbolic
 
-function f = ifourier(F, k, x)
+function f = ifourier(varargin)
+
+  syms F w x;
+  if (nargin == 1)
+    F=varargin{1};
+    w=symvar(F,1);
+
+  elseif (nargin == 2)
+    F=varargin{1};
+    w=symvar(F,1);
+    x=varargin{2};
 
-  if (nargin < 3)
-    syms k
-    warning('todo: check SMT for 2nd argument behavoir');
-  end
+  elseif (nargin == 3)
+    F=varargin{1};
+    w=varargin{2}; 
+    x=varargin{3};
 
-  cmd = { 'd = sp.inverse_fourier_transform(*_ins)'
-          'return d,' };
+  else
+    error('Wrong number of input arguments') 
+ 
+  endif
 
-  f = python_cmd (cmd, sym(F), sym(k), sym(x));
+  cmd = { 'sp.InverseFourierTransform._a = 0.5/S.Pi'
+          'sp.InverseFourierTransform._b = 1'
+          'F = sp.inverse_fourier_transform(*_ins)'
+          'return F,'};
+  f = python_cmd(cmd,F,w,x);
 
 end
 
 
-%! % tests in fourier.m
-%!assert(true)
+%!test
+%! syms x k
+%! f = exp(-x^2);
+%! F = fourier(f,x,k);
+%! g = ifourier(F,k,x);
+%! assert(isequal(f,g))
```

Comment:
is just copy-paste of the fourier.m ones?  How about testing that the calling forms without x,k work?  Actually I guess the doctests will get that as you've added it to the docs, so not necessary.  This is ok.